Classes/Service/FileGenerator.php

```diff
@@ -1,5 +1,4 @@
 <?php
-declare(strict_types=1);
```

Comment:
why that?

Reply:
Added it accidentally.
Should be added in all files in one commit...